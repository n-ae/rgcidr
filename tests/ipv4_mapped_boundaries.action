::ffff:192.168.0.0/112
