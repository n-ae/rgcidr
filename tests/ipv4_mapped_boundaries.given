::ffff:0.0.0.0
::ffff:0.0.0.1
::ffff:127.255.255.255
::ffff:128.0.0.0
::ffff:255.255.255.254
::ffff:255.255.255.255
::FFFF:0.0.0.0
::FFFF:255.255.255.255
0:0:0:0:0:ffff:0.0.0.0
0:0:0:0:0:ffff:255.255.255.255
0:0:0:0:0:FFFF:0.0.0.0
0:0:0:0:0:FFFF:255.255.255.255
0000:0000:0000:0000:0000:ffff:0.0.0.0
0000:0000:0000:0000:0000:ffff:255.255.255.255
::ffff:192.168.0.0
::ffff:192.168.0.255
::ffff:192.168.1.0
::ffff:192.168.255.255
::ffff:10.0.0.0
::ffff:10.255.255.255
::ffff:172.16.0.0
::ffff:172.31.255.255
::1
::2
2001:db8::1
fe80::1
