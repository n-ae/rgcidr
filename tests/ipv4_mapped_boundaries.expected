::ffff:192.168.0.0
::ffff:192.168.0.255
::ffff:192.168.1.0
::ffff:192.168.255.255
